ceremonies/src/tests.rs

```diff
@@ -1452,15 +1452,15 @@ fn endorse_newbie_fails_if_already_endorsed_in_previous_ceremony() {
 }
 
 #[test]
-fn endorse_newbie_fails_if_not_authorized() {
+fn endorse_newbie_fails_without_reputation() {
 	new_test_ext().execute_with(|| {
 		let cid = perform_bootstrapping_ceremony(None, 1);
 
 		let yran = account_id(&sr25519::Pair::from_entropy(&[8u8; 32], None).0);
 		let zoran = sr25519::Pair::from_entropy(&[9u8; 32], None).0;
 		assert_err!(
 			EncointerCeremonies::endorse_newcomer(Origin::signed(account_id(&zoran)), cid, yran),
-			Error::<TestRuntime>::AuthorizationRequired
```

Comment:
AuthorizationRequired is now an orphaned Error. Shall we remove it (I think there are other orphans by now)
This will mess up the indexes of the enum, but I think we always rely on metadatsa, so that shouldn't matter ?

Reply:
Yes, it should not matter. I believe it does not even matter for the rust-cli any more. I think we can remove it.